admin/services/pin.service.js

```diff
@@ -12,12 +13,18 @@ const pinService = {}
 /**
  * Get pupils with active pins
  * @param dfeNumber
- * @returns {Array}
+ * @param schoolId (optional, defaults to null)
+ * @returns {Promise<*>}
  */
-pinService.getPupilsWithActivePins = async (dfeNumber) => {
+pinService.getPupilsWithActivePins = async (dfeNumber, schoolId = null) => {
```

Comment:
second argument added as optional with a null as default

Reply:
Do you have tests to support these changes in this method?

Reply:
This just looks wrong.  Can we not choose one or the other please?